### PR TITLE
Facilitator 1920 app 2

### PR DESCRIPTION
**Pull request by clareconstantine**
(to be merged second)

- Can view and submit Facilitator 1920 applications
- App dashboard shows Facilitator 1920 applications
- Corresponding tests are updated to use 1920 applications

### CODE REVIEW

**Comment by clareconstantine** [comment]
[Begin setting up 2019-20 facilitator application template](https://trello.com/c/So57SbEB/764-begin-setting-up-2019-20-facilitator-application-template)

File: dashboard/app/models/pd/application/facilitator1920_application.rb [hunk 1]
```diff
@@ -51,5 +55,130 @@ def self.open?
     def check_idempotency
       Pd::Application::Facilitator1920Application.find_by(user: user)
     end
+
+    # memoize in a hash, per course
+    FILTERED_LABELS ||= Hash.new do |h, key|
+      labels_to_remove = (key == 'csf' ?
```
Comment (mehalshah) [inline dashboard/app/models/pd/application/facilitator1920_application.rb]: I think the parens are unnecessary

**Review by mehalshah — APPROVED** [review]
This looks great! Does it pass the facilitator application eyes test?

**Comment by clareconstantine** [comment]
I've been trying to get the test running locally so I can see what the problem is - when I look at the dashboard manually it looks fine to me so I'm not sure why it's failing yet.

**Pull request merged**